src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1659,6 +1659,7 @@ private function registerMessengerConfiguration(array $config, ContainerBuilder
             'before' => [
                 ['id' => 'add_bus_name_stamp_middleware'],
                 ['id' => 'dispatch_after_current_bus'],
+                ['id' => 'messenger.middleware.failed_message_processing_middleware'],
```

Comment:
why does this include the messenger.middleware prefix but the others don't?

Reply:
Good catch. It's optional - works both ways - but it should not be there for consistency.